.jscsrc

```diff
@@ -1,40 +0,0 @@
-{
```

Comment:
@patsissons, we can't remove this file. Also as a validation process. I added this is no accident, JSCS helps to have one code style and write code that's based one 1 style guide. I think we can't remove that! Now I see different variables naming, maybe there are another problems with code style.